consoles/sshXtermIPMI.pm

```diff
@@ -38,6 +38,8 @@ sub activate {
     push(@command, qw(sol activate));
     my $serial = $self->{args}->{serial};
     my $cstr   = join(' ', @command);
+    bmwqemu::diag "Try to deactivate IPMI SOL before activate";
+    eval { $self->backend->ipmitool("sol deactivate"); };
```

Comment:
> Why isn't this also using `reset`?

There is no any activate console recording at the test beginning, reset() using $self->{activated} to detect console status first, if use it in this scenario, reset() could not reach "sol deactivate". 

Here change is for safety to avoid any un-deactivated SOL console breaks test at beginning. 

For example, some one directly access SOL but forget deactivating it after work complete. openQA triggered test job on this machine, the test job will not be broken with this changes. This scenario often happens on sharing machine between debugging and openQA job.

Reply:
It would still be nice to catch the error and only suppress the expected error case instead of ignoring any kind of error.

Reply:
As Martchus said. Calling `sol deactivate` in the case of an already deactivated SOL will also trigger an error "Info: SOL payload already de-activated" which should be avoided

Reply:
Thanks for your comments! 
Deactivating SOL on a deactivated SOL machine,  "Info: SOL payload already de-activated" is as expected and harmless, it will not break openQA test run, so that is just only for safety.
The goal of this change is un-block openQA test run on a machine with SOL activate, having this action is better than not.

Reply:
ok but please still see what Martchus stated. The generic `eval` with ignoring any error should be avoided

Reply:
> ok but please still see what Martchus stated. The generic `eval` with ignoring any error should be avoided
Thanks for your comments! Updated code to check if IPMI response is expected or not after 'eval', would you please help to review? Thank you again!

Reply:
The error handling looks good now.

Reply:
> The error handling looks good now.

Thank you!